src/tracks/ui/BrushHandle.cpp

```diff
@@ -282,31 +285,44 @@ UIHandle::Result BrushHandle::Drag
    mMostRecentX = x;
    mMostRecentY = y;
```

Comment:
I think mMostRecentX, mMostRecentY, mAutoScrolling, and the TimerHandler class are all legacies of SelectHandle that you don't need.  There are still unnecessary things to remove from BrushHandle.cpp.

Reply:
`mMostRecentX` and `mMostRecentY` is used for drawing the outline of the brush tool, while I would like to keep `mAutoScrolling`, in case we decide to implement it in the future, since potentially we can drag near the edge of the track and expect scrolling to happen.
Will remove the `TimerHandler`

Reply:
`mMostRecentX` and `mMostRecentY` is used for drawing the outline of the brush tool.
Will remove the `TimerHandler` and `mAutoScrolling`